package.json

```
@@ -15,13 +15,13 @@
     "url": "https://github.com/standard/standard/issues"
   },
   "dependencies": {
-    "eslint": "~7.18.0",
-    "eslint-config-standard": "16.0.3",
-    "eslint-config-standard-jsx": "10.0.0",
-    "eslint-plugin-import": "~2.25.3",
-    "eslint-plugin-node": "~11.1.0",
-    "eslint-plugin-promise": "~5.1.1",
-    "eslint-plugin-react": "~7.27.0",
+    "eslint": "^8.7.0",
+    "eslint-config-standard": "^17.0.0-0",
+    "eslint-config-standard-jsx": "^10.0.0",
+    "eslint-plugin-import": "^2.25.4",
+    "eslint-plugin-node": "^11.1.0",
```

```suggestion
    "eslint-plugin-n": "^14.0.0",
```

it's a maintained fork of the plugin.

---

👍 Yeah, its the one required by `"eslint-config-standard": "^17.0.0-0"`: https://github.com/standard/eslint-config-standard/blob/3bc8b18ebc3925f020b13af4a0226e7444878528/package.json#L16